beep/features/featurizer_helpers.py

```diff
@@ -886,3 +887,57 @@ def get_fractional_quantity_remaining(
     )
     summary_diag_cycle_type.columns = ["cycle_index", "fractional_metric"]
     return summary_diag_cycle_type
+
+
+def get_fractional_quantity_remaining_nx(
+    processed_cycler_run, metric="discharge_energy", diagnostic_cycle_type="rpt_0.2C"
+):
+    """
+    Similar to get_fractional_quantity_remaining()
+    Determine relative loss of <metric> in diagnostic_cycles of type <diagnostic_cycle_type> after 100 regular cycles
+    Also returns value of 'x', the discharge throughput passed by the first diagnostic
+    and the value 'n' at each diagnostic
+
+    Args:
+        processed_cycler_run (beep.structure.ProcessedCyclerRun): information about cycler run
+        metric (str): column name to use for measuring degradation
+        diagnostic_cycle_type (str): the diagnostic cycle to use for computing the amount of degradation
+
+    Returns:
+        a dataframe with cycle_index and corresponding degradation relative to the first measured value
```

Comment:
Update this to reflect what the method return (dataframe with cycle_index, fractional_metric, x, n)